astro/src/content/docs/lifecycle/authenticate-users/contextual-multi-factor.mdx

```diff
@@ -0,0 +1,107 @@
+import MfaDecisionTree from 'src/diagrams/docs/lifecycle/authenticate-users/mfa-decision-tree.astro';
+import TenantMfaDecisionTree from 'src/diagrams/docs/lifecycle/authenticate-users/tenant-mfa-decision-tree.astro';
+import ApplicationMfaDecisionTree from 'src/diagrams/docs/lifecycle/authenticate-users/app-mfa-decision-tree.astro';
+
+## Overview
+
+FusionAuth applies multi-factor authentication based on a number of configuration and situational behaviors. In this document you'll learn about the way FusionAuth automatically applies MFA during the login process.
+
+If you are looking for guidance on implementing MFA, please [review the MFA guide](/docs/lifecycle/authenticate-users/multi-factor-authentication).
+
+## MFA Inside The Authentication Process
+
+There are four major considerations for whether MFA is applied during the login process.
+
+* What is the MFA policy for the tenant or application? It can be either `Disabled`, `Enabled`, or `Required`.
+* Does the user have MFA enabled? If this is not the case, then an MFA challenge is never triggered, but the user may be prompted to set up MFA, based on the tenant or application MFA policy.
+* Did the user log in using an Identity Provider such as Google or OIDC? In this case, MFA is never required for the user.
+* Has the device passed appropriate contextual checks such as being on a known device?
+
+Here's a high level diagram of the logic:
+
+<MfaDecisionTree alt="High level diagram of MFA logic."/>
+
+### Contextual Checks
+
+The contextual checks are based on attributes of the request evaluated every time an authentication happens and and include:
+
+* Has this device been seen before?
+* Has this user been seen before on this device?
+* Is there a suspicious login detected for this authentication attempt?
+
+The duration of trust of the device can be configured with the `twoFactorTrustIdTimeToLiveInSeconds` tenant configuration value.
+
+The goal of this contextual check is to challenge the user for another factor of authentication whenever FusionAuth determines the risk of incorrect access is higher than the user hassle.
+
+Depending on [your license](#license-limitations), you can configure MFA policies at both the tenant and application level. Let's take a look at each of these in more depth.
+
+### Tenant Level MFA Configuration
+
+Here's a diagram of the MFA challenge logic when the tenant has a policy for MFA.
+
+<TenantMfaDecisionTree alt="Diagram of tenant MFA logic."/>
+
+### Application Level MFA Configuration
+
+Here's a diagram of MFA challenge logic when the application has a policy for MFA. Application policies always supersede the tenant policy.
+
+<ApplicationMfaDecisionTree alt="Diagram of application MFA logic."/>
+
+## License Limitations
+
+Not all plans support all MFA features. The plan you are on affects the MFA options available to you and your users.
+
+<EnterprisePlanBlurb />
```

Comment:
The way these blurbs read with a bulleted list isn't quite right, but I'm not sure it's worth reworking for this one case.

They both say "This feature is only available" followed by a list of multiple items.